### PR TITLE
Add account bio to account admin view

### PR DESCRIPTION
**Pull request by ClearlyClaire**
image

### CODE REVIEW

File: app/views/admin/accounts/show.html.haml [hunk 1]
```diff
@@ -1,8 +1,14 @@
 - content_for :page_title do
   = @account.acct
 
+- proofs = @account.identity_proofs.active
+- fields = @account.fields
```
Comment (ykzts) [inline app/views/admin/accounts/show.html.haml]: need?

Comment (ThibG) [reply app/views/admin/accounts/show.html.haml]: It's not needed. Sorry, this was leftover from a previous attempt at that PR!
Removed it

**Review by ykzts — APPROVED** [review]

**Comment by ThibG** [comment]
@Gargron ping

**Comment by Gargron** [comment]
It's really stretched out to the point it's hard to read, can we do something about that? I'm not sure it's necessary since I always click to just open the real profile

**Comment by ThibG** [comment]
Good point, I'll try thinking about a better layout

**Comment by ThibG** [comment]
Updated so it looks like this:

image

**Comment by Sylvhem** [comment]
Yes! Yes, pretty please. I constantly have to open users' profiles to see if their bio is not full of sponsored links. This would save me so much time and half the clicks.

**Pull request merged**